pontoon/base/models.py

```diff
@@ -232,6 +307,24 @@ def serialize(self):
             'obsolete': self.obsolete,
         }
 
+    def has_changed(self, locale_code):
```

Comment:
Would these querysets be evaluated only once instead of two times if you moved the .exists() piece right after .filter()? https://docs.djangoproject.com/en/1.8/ref/models/querysets/#when-querysets-are-evaluated